kurtosis-devnet/pkg/kurtosis/api/fake/fake.go

```diff
@@ -31,6 +33,27 @@ func (f *KurtosisContext) GetEnclave(ctx context.Context, name string) (interfac
 	return f.EnclaveCtx, nil
 }
 
+func (f *KurtosisContext) GetEnclaveStatus(ctx context.Context, name string) (string, error) {
+	if f.GetErr != nil {
+		return "", f.GetErr
+	}
+	return "running", nil
```

Comment:
better to have a proper type and associated constants for the various values of status.